sql-plugin/src/main/scala/com/nvidia/spark/rapids/Plugin.scala

```diff
@@ -111,6 +111,31 @@ object RapidsPluginUtils extends Logging {
     }
   }
 
+  private def detectMultipleJar(propName: String, jarName: String, 
+      complainFun: (Boolean, String) => Unit): Unit = {
+    val classloader = ShimLoader.getShimClassLoader()
+    val rapidsJarURLs = classloader.getResources(propName).asScala.toList
```

Comment:
I see with driver-class-path we might end up with duplicate entries of the same URL. Let us dedupe here

```suggestion
    val rapidsJarURLs = classloader.getResources(propName).asScala.toSet
```